### DISCUSSION TITLE
Swap Textures/materials - diffuse

### DISCUSSION THREAD

**Discussion by Arumugamhns** [google/model-viewer#2313] [Q&A]
Description

Need to swap textures. I have a car glb file. I need to change the exterior color alone. i have worked using the modelviewer examples but it change whole vehicle including interior (refer attached image). kindly suggest how to texture or change the diffuse for selected object 
![1](https://user-images.githubusercontent.com/60291301/115710469-033ed680-a390-11eb-976e-a38bc3d04b04.JPG)
![blue](https://user-images.githubusercontent.com/60291301/115710473-04700380-a390-11eb-97ef-f42ebe37062b.JPG)

Version
- model-viewer: v1.6.0

AR
Working fine in android but not working in iOS AR

**Comment by elalish** [comment]
How many materials is your glTF broken into? If it's all one material, then there's nothing we can do. Otherwise you just have to choose the one you want from our list. Also consider the glTF variants extension, which we support to make this easier: https://modelviewer.dev/examples/scenegraph/#variants
